### PR TITLE
Fix NRE in MethodParameterLookup

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**

### CODE REVIEW

File: analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpMethodParameterLookup.cs [hunk 1]
```diff
@@ -31,12 +31,12 @@ internal class CSharpMethodParameterLookup : AbstractMethodParameterLookup<Argum
         }
 
         public CSharpMethodParameterLookup(ArgumentListSyntax argumentList, SemanticModel semanticModel)
```
Comment (csaba-sagi-sonarsource) [inline analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpMethodParameterLookup.cs:33]: Maybe these constructors could be single line.

File: analyzers/src/SonarAnalyzer.VisualBasic/Helpers/VisualBasicMethodParameterLookup.cs [hunk 2]
```diff
@@ -26,12 +26,12 @@ namespace SonarAnalyzer.Helpers
     internal class VisualBasicMethodParameterLookup : AbstractMethodParameterLookup<ArgumentSyntax>
     {
         public VisualBasicMethodParameterLookup(ArgumentListSyntax argumentList, IMethodSymbol methodSymbol)
```
Comment (csaba-sagi-sonarsource) [inline analyzers/src/SonarAnalyzer.VisualBasic/Helpers/VisualBasicMethodParameterLookup.cs:28]: Same here, if you want you can move the constructors to single line.

**Review by csaba-sagi-sonarsource — APPROVED** [review]
LGTM

File: analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpMethodParameterLookup.cs (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply analyzers/src/SonarAnalyzer.CSharp/Helpers/CSharpMethodParameterLookup.cs:33]: The 2nd one has some logic that should be on a separate line. So I'll keep the 1st one the same for consitency.

**Pull request merged**